config/initializers/foreman.rb

```diff
@@ -19,6 +19,7 @@
   end
 
   Setting.descendants.each(&:load_defaults)
+  Setting.validate_all
```

Comment:
This will work OK on startup, but how does validation work from the settings controllers?  If I change the value of these things, there's no validation performed until the next time the server starts, which is going to be problematic.

Validation makes most sense at the model level I think, but validating at any time over multiple settings is going to cause problems for users who can only change one thing at a time.